lib/r10k/action/deploy/display.rb

```diff
@@ -9,25 +9,46 @@ class Display < R10K::Action::Base
 
         include R10K::Action::Deploy::DeployHelpers
 
+        def initialize(opts, argv, settings)
+          super
+
+          @settings = @settings.merge({
+            overrides: {
+              environments: {
+                preload_environment: @fetch,
+                requested_environments: @argv
```

Comment:
I see that we didn't do any correction of this before, like we do in [deploy/environment](https://github.com/puppetlabs/r10k/blob/92df0c2e6fb762e7acc16e6f50653e63c7dcafa6/lib/r10k/action/deploy/environment.rb#L29). Is that intentional, or an oversight?

Reply:
I believe that's an oversight.